### PR TITLE
Fix bindless buffer overruns

### PR DESCRIPTION
**Pull request by Joshua-Ashton**

### CODE REVIEW

File: libs/vkd3d/command.c [hunk 1]
```diff
@@ -2078,7 +2078,7 @@ static void d3d12_command_list_clear_attachment_deferred(struct d3d12_command_li
 
     /* If necessary, combine with previous clear so that e.g. a
      * depth-only clear does not override a stencil-only clear. */
-    clear_state->attachment_mask |= 1u << attachment_idx;
+    clear_state->attachment_mask |= 1llu << attachment_idx;
```
Comment (HansKristian-Work) [inline libs/vkd3d/command.c:2081]: This seems a bit curious. Do we support more than 32 attachments in D3D12/Vulkan?

Comment (doitsujin) [reply libs/vkd3d/command.c:2081]: no, maximum is 8.

Comment (Joshua-Ashton) [reply libs/vkd3d/command.c:2081]: Can we make the attachment mask a `uint32_t` then?

Comment (Joshua-Ashton) [reply libs/vkd3d/command.c:2081]: I have dropped that commit for now...

File: build-win32.txt [hunk 2]
```diff
@@ -3,7 +3,6 @@ c = 'i686-w64-mingw32-gcc'
 cpp = 'i686-w64-mingw32-g++'
 ar = 'i686-w64-mingw32-ar'
 strip = 'i686-w64-mingw32-strip'
-cmake = 'i686-w64-mingw32-cmake'
```
Comment (HansKristian-Work) [inline build-win32.txt:6]: Tried rebasing? This commit should already be on master.

Comment (Joshua-Ashton) [reply build-win32.txt:6]: IDK how this got caught up in this series...

Will do

Comment (Joshua-Ashton) [reply build-win32.txt]: Resolved.

**Pull request merged**